ALUM/ALUM/Views/SignUpPage/SignUpSetUp.swift

```diff
@@ -0,0 +1,104 @@
+                    }
+                    .padding(.bottom, 32)
+                    .padding(.top, 8)
+
+                    InputValidationComponent(text: $viewModel.account.email, componentName: Text("Email: ")
+                        .font(.custom("Metropolis-Regular", size: 16)), labelText: "Email", showCheck: true, functions: viewModel.emailFunc)
+                        .padding(.bottom, 32)
+
+                    InputValidationComponent(text: $viewModel.account.name, componentName: Text("Name: ").font(.custom("Metropolis-Regular", size: 16)), labelText: "Name")
+                        .padding(.bottom, 32)
+
+                    InputValidationComponent(text: $viewModel.account.password, componentName: Text("Password: ")
+                        .font(.custom("Metropolis-Reegular", size: 16)), labelText: "Password", isSecured: true, showEye: true, showCheck: true, functions: viewModel.passFunc)
+                        .padding(.bottom, 32)
+
+                    InputValidationComponent(
+                        text: $viewModel.passwordAgain,
+                        componentName: Text("Confirm Password: ").font(.custom("Metropolis-Regular", size: 16)),
+                        labelText: "Password",
+                        isSecured: true,
+                        showEye: true,
+                        functions:viewModel.passAgainFunc
+                    )
+                    .padding(.bottom, 32)
+                    
+                    if viewModel.account.email != "" && viewModel.account.password != "" &&
+                        viewModel.account.name != "" && viewModel.passwordAgain != "" &&
+                        SignUpPageViewModel.Functions.IUSDEmail(viewModel.account.email).0 &&
+                        SignUpPageViewModel.Functions.EightChars(viewModel.account.password).0 &&
+                        SignUpPageViewModel.Functions.OneNumber(viewModel.account.password).0 &&
+                        SignUpPageViewModel.Functions.SpecialChar(viewModel.account.password).0 &&
+                        viewModel.account.password == viewModel.passwordAgain
+                    {
+                        NavigationLink(destination: SignUpJoinAs(viewModel: viewModel), label: {Text("Continue")})
+                        .buttonStyle(FilledInButtonStyle(disabled: false))
+                        .padding(.leading, 16)
+                        .padding(.trailing, 16)
+                        .padding(.bottom, 32)
+                    } else {
+                        Button("Continue") {
+                            viewModel.checkPasswordSame()
+                        }
+                        .buttonStyle(FilledInButtonStyle(disabled: true))
+                        .padding(.leading, 16)
+                        .padding(.trailing, 16)
+                        .padding(.bottom, 32)
+                    }
+
+                }
+            }
+            .edgesIgnoringSafeArea(.all)
+            .padding(.top, 62)
```

Comment:
action - same comment from SignUpJoinAs to change the structure of the Z, V, H stacks.